### DISCUSSION TITLE
未来会出屏蔽up主的功能吗

### DISCUSSION THREAD

**Discussion by guodayuyu** [the1812/Bilibili-Evolved#1877] [功能建议 / Ideas]
**bilibili app现在刷视频已经可以设置不喜欢的内容及up主，而这一方面在电脑端还很空缺，主要是有很多up主上传很多视频而且均毫无营养，导致浏览相关分区热度排序时体验非常不佳，希望将来能出个屏蔽up主视频的功能**
    类似下面这个作者https://github.com/hanFengSan/BilibiliCleaner，遗憾的是这个作者现在已经不更新了....

**Comment by Geeyun-JY3** [comment]
请先善用搜索：#613、#1279、#1486、#1705、#1733
